docs_user/modules/proc_stopping-openstack-services.adoc

```diff
@@ -183,3 +161,5 @@ for i in {1..3}; do
     fi
 done
 ----
++
+If the status of each service is `OK`, then the services stopped successfully.
```

Comment:
Question for SME maybe. How to check status? $ systemctl status <service-name> ? If so, should status be "Inactive" if it's stopped?

Reply:
CC: @jistr

Reply:
I guess the gist is that we process what the low level commands print and then just distill it into "OK" / "ERROR" messages (see the snippet). So i think the sentence we're discussing is correct.